### PR TITLE
Add ShellCheck engine definition

### PR DESCRIPTION
**Pull request by pbrisbin**
/cc @codeclimate/review

### CODE REVIEW

File: config/engines.yml [hunk 1]
```diff
@@ -249,6 +249,16 @@ rubymotion:
   enable_regexps:
   default_ratings_paths:
     - "**.rb"
+
```
Comment (wfleming) [inline config/engines.yml]: Nit, but right now we don't have spaces between engine definitions. Seems like a nice addition to me, but if we're going to do it I think we should do all of them at once.

File: config/engines.yml [hunk 2]
```diff
@@ -249,6 +249,16 @@ rubymotion:
...
+shellcheck:
+  image: codeclimate/codeclimate-shellcheck
+  description: A static analysis tool for shell scripts
```
Comment (wfleming) [inline config/engines.yml]: :lipstick: description fields should be full sentences with periods at the end.

File: config/engines.yml (hunk 1 above)
Comment (pbrisbin) [inline config/engines.yml]: My bad, I thought I saw space between nearby ones. I was seeing things.

File: config/engines.yml (hunk 2 above)
Comment (pbrisbin) [inline config/engines.yml]: I can add the period, are you also saying I need to make this a grammatically correct sentence? "ShellCheck is a..."

File: config/engines.yml [hunk 3]
```diff
@@ -249,6 +249,16 @@ rubymotion:
   enable_regexps:
   default_ratings_paths:
     - "**.rb"
+
+shellcheck:
```
Comment (wfleming) [inline config/engines.yml]: I think this should be sorted after `scss-lint`, shouldn't it?

Comment (pbrisbin) [inline config/engines.yml]: I matched builder's assuming that one was sorted; guess not. Will fix here at least.

File: config/engines.yml (hunk 2 above)
Comment (wfleming) [inline config/engines.yml]: Good point. Other descriptions in the file are currently fragments, so for now I think just adding the period is fine, and we can wait for @scriberty to decide it's bad & needs changing.

File: config/engines.yml (hunk 3 above)
Comment (wfleming) [inline config/engines.yml]: Oops. Builder might have been my bad, sorry.

**Comment by wfleming** [comment]
LGTM

**Pull request merged**